### PR TITLE
fix #341 Send double values with 0 values in decimals

### LINKED ISSUE #341: [Bug Report] Fields with floating values at 0 do not save changes
## Bug report
Fields that handle values of type Double, when they have values with decimals in .0 are not saved when creating, or updating a record in a window.

#### Steps to reproduce

1. Open a window containing fields that handle decimal numbers.
2. Select a record.
3. In the decimal value field, set any value with decimals to zero.
4. Go to the context menu and select the action update.

The modified number fields will return to the previous value.

#### Screenshot or Gif
Peek 17-02-2020 18-51

#### Expected behavior
You are expected to store values of the type double whose value is equal to zero.

#### Additional context

However, if you store values whose decimal places are greater than 0, for example 0.00001

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactor
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, not the `master` branch
- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your `master` branch!
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number): fix #341 

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

## Description

How the fields that handle floating numbers are displayed store the fields with decimal values equal to 0

NOTE: This PR depends on this change in the grpc client library https://github.com/erpcya/gRPC-Data-Client/pull/52/

### Screenshots

 **Before this PR**
Peek 17-02-2020 18-51

**After this PR**
Peek 17-02-2020 18-52

### CODE REVIEW

File: src/components/ADempiere/Field/references.js [hunk 1]
```diff
@@ -1,235 +1,274 @@
...
+    isSupported: false,
     description: 'Color element',
+    valueType: 'STRING',
     alias: ['Color']
   },
   {
     id: 37,
     type: 'FieldNumber',
-    support: true,
+    isSupported: true,
```
Comment (yamelsenih) [inline src/components/ADempiere/Field/references.js]: Please keep it without changes and add other PR as minor change

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/Field/FieldNumber.vue [hunk 2]
```diff
@@ -24,6 +24,9 @@
 import { fieldMixin } from '@/components/ADempiere/Field/FieldMixin'
 import { FIELDS_FLOATS } from '@/components/ADempiere/Field/references'
 
+/**
+ * TODO: Add change switch decimal point (.) or decimal comma (,)
```
Comment (yamelsenih) [inline src/components/ADempiere/Field/FieldNumber.vue]: Please remove it

File: src/components/ADempiere/Field/references.js [hunk 3]
```diff
@@ -319,6 +358,6 @@ export const FIELD_READ_ONLY_FORM = [
   }
 ]
 
-export const FIELDS_FLOATS = ['Amount', 'Costs+Prices', 'Number']
+export const FIELDS_FLOATS = ['Amount', 'Costs+Prices', 'Number', 'Quantity']
```
Comment (yamelsenih) [inline src/components/ADempiere/Field/references.js]: Change it to FIELDS_DECIMALS

File: src/utils/ADempiere/dictionaryUtils.js [hunk 4]
```diff
@@ -160,7 +160,7 @@ export function generateField({
     isSOTrxMenu,
     // displayed attributes
     componentPath: componentReference.type,
-    isSupport: componentReference.support,
+    isSupported: componentReference.support,
```
Comment (yamelsenih) [inline src/utils/ADempiere/dictionaryUtils.js]: remove this change

File: src/store/modules/ADempiere/windowControl.js [hunk 5]
```diff
@@ -187,27 +187,33 @@ const windowControl = {
...
+          columnName: itemAttribute.columnName,
+          valueType: itemAttribute.valueType
+        })
       })
 
       commit('addInCreate', {
         containerUuid,
         tableName,
-        attributesList: finalAttributes
+        attributesList
```
Comment (yamelsenih) [inline src/store/modules/ADempiere/windowControl.js:210]: don't touch variables

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/Field/references.js [hunk 6]
```diff
@@ -319,6 +358,6 @@ export const FIELD_READ_ONLY_FORM = [
   }
 ]
 
-export const FIELDS_FLOATS = ['Amount', 'Costs+Prices', 'Number']
+export const FIELDS_DECIMALS = ['Amount', 'Costs+Prices', 'Number', 'Quantity']
 
 export const FIELDS_QUANTITY = ['Amount', 'Costs+Prices', 'Integer', 'Number', 'Quantity']
```
Comment (yamelsenih) [inline src/components/ADempiere/Field/references.js:363]: What is it?
for integer only existe **Integer** field type

File: src/store/modules/ADempiere/windowControl.js (hunk 5 above)
Comment (EdwinBetanc0urt) [reply src/store/modules/ADempiere/windowControl.js:210]: is part of the change

**Pull request merged**